.github/workflows/ci-test-ginkgo.yml

```diff
@@ -88,7 +88,8 @@ jobs:
         env:
           GOPATH: /home/runner/go
 
-      - uses: codecov/codecov-action@v3
+      - uses: actions/upload-artifact@v3
         if: ${{ always() }}
         with:
-          files: ./KubeArmor/gover.coverprofile
+          name: coverage
+          path: ./KubeArmor/gover.coverprofile
```

Comment:
I think this won't be generate for now, so it's fine if it's empty. Let's come back to this later and might as well remove the dependency for it.